Cargo.toml

```diff
@@ -33,6 +33,7 @@ serde_test = "1.0.19"
 
 [features]
 default = ["std"]
+nightly = []
```

Comment:
I don't think we need this for this atm.. see the other comment below

Reply:
@kinggoesgaming It is not working when I remove ```nightly = []``` and use ```#![cfg_attr(nightly, feature(const_fn))]```

Reply:
tested it locally.. its works for me... have you tried `cargo +nightly build`? it needs that `+nightly` bit... if you get something like toolchain not found error... run `rustup install nightly` and try again

Reply:
travis passes as well on nightly means it is working

Reply:
@kinggoesgaming @Dylan-DPC Why is it only ```test_uuid_macro_build```test passed? The ```test_const_uuid_macro_build``` should also be passed.
![image](https://user-images.githubusercontent.com/16131917/39669058-8402c79a-5113-11e8-8096-601b063fc289.png)

Reply:
hmm its not even being detected/run, which is indeed weird... 

have you pushed these changes to your fork?

Reply:
@kinggoesgaming It's didn't work as expectedly, So I haven't pushed it yet.

Reply:
@kinggoesgaming I pushed it on [uuid_macro/nightly-uuid](https://github.com/uuid-rs/uuid_macro/tree/nightly-uuid)

Reply:
based on the discussion in #220 we can go back to this. so this is fine